seacrowd/sea_datasets/visim_400/visim_400.py

```diff
@@ -0,0 +1,181 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+ViSim-400 is a Vietnamese dataset of semantic relation \
+    pairs for evaluation of models that reflect the \
+    continuum between similarity and relatedness.
+"""
+
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses
+
+_CITATION = """\
+@inproceedings{nguyen-etal-2018-introducing,
+    title = "Introducing Two {V}ietnamese Datasets for Evaluating Semantic Models of (Dis-)Similarity and Relatedness",
+    author = "Nguyen, Kim Anh  and
+      Schulte im Walde, Sabine  and
+      Vu, Ngoc Thang",
+    editor = "Walker, Marilyn  and
+      Ji, Heng  and
+      Stent, Amanda",
+    booktitle = "Proceedings of the 2018 Conference of the North {A}merican Chapter of the Association for Computational Linguistics: Human Language Technologies, Volume 2 (Short Papers)",
+    month = jun,
+    year = "2018",
+    address = "New Orleans, Louisiana",
+    publisher = "Association for Computational Linguistics",
+    url = "https://aclanthology.org/N18-2032",
+    doi = "10.18653/v1/N18-2032",
+    pages = "199--205",
+    abstract = "We present two novel datasets for the low-resource language Vietnamese to assess models of semantic similarity: ViCon comprises pairs of synonyms and antonyms across word classes, thus offering data to distinguish between similarity and dissimilarity. ViSim-400 provides degrees of similarity across five semantic relations, as rated by human judges. The two datasets are verified through standard co-occurrence and neural network models, showing results comparable to the respective English datasets.",
+}
+"""
+
+_DATASETNAME = "visim_400"
```

Comment:
We follow the dataset naming from datasheet [#579](https://github.com/SEACrowd/seacrowd-datahub/issues/579). Thus, this should be `visim400`, including the file and folder name.

Reply:
Solved by:
- https://github.com/SEACrowd/seacrowd-datahub/pull/608/commits/0ae64a320743b8467e94b8873b1233382a69d417
- https://github.com/SEACrowd/seacrowd-datahub/pull/608/commits/25bd2a1edc59dc3248443fb40ec36ee99affb141
- https://github.com/SEACrowd/seacrowd-datahub/pull/608/commits/25ac8fd3c1fbf836a80af1c4fd46df7a226ab7fd
- https://github.com/SEACrowd/seacrowd-datahub/pull/608/commits/1d9dea4affcf6cf177bd2598f7b822df158b09ad
- https://github.com/SEACrowd/seacrowd-datahub/pull/608/commits/9d9ae1b516d2de55b37dd045b18b5f5b9fe2641d
- https://github.com/SEACrowd/seacrowd-datahub/pull/608/commits/d6369a3eeaa4bdf5ca4f65d04994ae68ff4ca032

Because of that new folder name, I can't directly annotate your comment. I will comment with the commit.